libs/azure-storage/pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[tool.poetry]
+name = "azure-storage"
+version = "0.1.0"
+description = "Azure Storage library for LangChain Azure"
+authors = []
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
+langchain-core = "^0.3.0"
+typing-extensions = "^4.9.0"
+unstructured = "^0.8.0"
+azure-storage-blob = "^12.16.0"
+azure-identity = "^1.16.0"
+
+[tool.poetry.group.test.dependencies]
+pydantic = "^2.9.2"
+pytest = "^7.4.3"
```

Comment:
Let's try to cut down the number of dependencies that we have in each of these groups for now unless we know that we need them. For this one, let's try removing everything but `pytest` and see if CI passes.